Druid/DRPrimitiveCompilerCompilerCollector.class.st

```diff
@@ -0,0 +1,41 @@
+"
+This is a helper class to collect primitive method which couldn't be compiled by Druid.
+
+"
+Class {
+	#name : #DRPrimitiveCompilerCompilerCollector,
+	#superclass : #DRPrimitiveCompilerCompiler,
```

Comment:
A collector shouldn't be another object? Instead of subclass?

Maybe this should be move to the any compilation builder or pushed to the superclass

Reply:
Why couldn't be a subclass of DRPrimitiveCompilerCompiler if reuses behavior?